web/src/helpers/i18n.js

```diff
@@ -56,9 +58,9 @@ i18n
   .use(initReactI18next)
   .init({
     fallbackLng: 'en',
-    debug: isProduction() ? false : true,
+    debug: isProduction() ? true : true,
```

Comment:
Let's keep debugging off
```suggestion
    debug: isProduction() ? false : true,
```